caramel/config.py

```diff
@@ -162,19 +162,22 @@ def get_db_url(arguments=None, settings=None, required=True):
 
 def get_log_level(argument_level, logger=None, env=None):
     """Calculates the highest verbosity(here inverted) from the argument,
-    environment and root, capping it to between 0-3, returning the log level"""
+    environment and root, capping it to between logging.DEBUG(10)-logging.ERROR(40),
+    returning the log level"""
 
     if env is None:
         env = os.environ
-    env_level = int(env.get("CARAMEL_LOG_LEVEL", 0))
+    env_level = LOG_LEVEL[env.get("CARAMEL_LOG_LEVEL", "ERROR").upper()]
```

Comment:
I'd split this one into two statements with a name, as the KeyError otherwise becomes an ugly exception when a user passes in something not quite expected here. By splitting the "env.get" out, the error becomes easier to read when that happens:

export CARAMEL_LOG_LEVEL="WARN" 
```
env_level_name = env.get("CARAMEL_LOG_LEVEL", "ERROR").upper()
env_level = LOG_LEVEL[env_level_name]
```
exception raised..  KeyError (   not found, blah blah ....)     

Note, I'm not saying to catch that kind of error, just to organize the code so it's easy for the user to read the backtrace when they get that error.